### DISCUSSION TITLE
Install Script for anyone that needs it :)

### DISCUSSION THREAD

**Discussion by MrAshCreates** [GitoxideLabs/gitoxide#1394] [Ideas]
You can check out [here](https://github.com/MrAshCreates/Gitoxide-Installer) on my github account. Feel free to steal it from me.

**Comment by Byron** [comment]
Very practical, thanks for sharing!

**Reply by MrAshCreates** [reply]
sure thing, I was wanting to help support the project but I don't always know how so since I had trouble figuring out how to install decided to make the trouble disappear :)

**Reply by MrAshCreates** [reply]
If there is anyway I can help contribute let me know and I will try my best to learn, help, etc

**Reply by NobodyXu** [reply]
cargo-binstall/cargo-quickinstall could install gitoxide from quickinstall, which provides pre-built binaries for many popular rust crates.

**Reply by Byron** [reply]
> If there is anyway I can help contribute let me know and I will try my best to learn, help, etc

Thanks! The easiest way to find things to do is to look at the `crate-status.md` and find unticked check-boxes that interest you. Some of these might be relatively easy to get started with, which is something I can help you with once presented with an option or two.
